simapp/v2/app_di.go

```diff
@@ -92,7 +91,7 @@ func NewSimApp[T transaction.Tx](
 	logger log.Logger,
 	viper *viper.Viper,
 ) *SimApp[T] {
-	viper.Set(serverv2.FlagHome, DefaultNodeHome) // TODO possibly set earlier when viper is created
+	//viper.Set(serverv2.FlagHome, DefaultNodeHome) // TODO possibly set earlier when viper is created
```

Comment:
Why is it commented?

Reply:
this was the only way to enforce another folder than the home one. For the tests in order to be able to clear it in the end I want to use t.TempDir().

I even wonder if having the viper.set there if it is correct.

Reply:
It was for avoiding it doing it in newApp (in root.go or somewhere there). We should move it there then

Reply:
bump on this